src/services/web3.js

```diff
@@ -573,8 +574,76 @@ function sendTransaction(
   }
 }
 
+function getGasPrice(network: Network): Promise<BigNumber> {
+  const {
+    rpcaddr,
+    rpcport,
+    ssl,
+  } = network
+
+  const baseProps = {
+    rpcaddr,
+    rpcport,
+    ssl,
+  }
+
+  return jibrelContractsApi.eth.getGasPrice(baseProps)
```

Comment:
`baseProps` is useless veriable here

Reply:
it is necessary to validate that gasPrice is BigNumber and convert it in string

Reply:
1. Fixed
2. This check will be in saga. Let's move all error messages from network layer to sagas, we need to translate them